### PR TITLE
Implemented static UI for PO review page(#85zrgy1rw)

### LINKED ISSUE #117: Implement static UI of PO review page for multiple PO's
## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #117 

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2022-12-28 12-32-59

Screenshot from 2022-12-28 16-40-05

Screenshot from 2022-12-28 16-40-16

Screenshot from 2022-12-28 16-40-30

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/import#contribution-guideline)

### CODE REVIEW

**Comment by shashwatbangar** [comment]
On PODetail page, PO2 is sticking to left border.
Rest UI looks good to me.

**Comment by disha1202** [comment]
> On PODetail page, PO2 is sticking to left border. Rest UI looks good to me.
Fixed and added some more UI elements.

File: src/components/BulkAdjustmentModal.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<template>
+    <ion-header>
```
Comment (ymaheshwari1) [inline src/components/BulkAdjustmentModal.vue:2]: Improve indentation throughout.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/BulkAdjustmentModal.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Bulk adjustment") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
```
Comment (dt2patel) [inline src/components/BulkAdjustmentModal.vue:13]: Remove ion padding

File: src/components/DateTimeParseErrorModal.vue [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Date time parse error") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
```
Comment (dt2patel) [inline src/components/DateTimeParseErrorModal.vue:13]: Remove padding here and everywhere else.

**Review by dt2patel — CHANGES_REQUESTED** [review]
Need to fix indentation and review element selector practices. I think you can do one round of improvements yourself @disha1202

File: src/components/DateTimeParseErrorModal.vue (hunk 3 above)
Comment (disha1202) [reply src/components/DateTimeParseErrorModal.vue]: Removed.

**Comment by disha1202** [comment]
> Need to fix indentation and review element selector practices. I think you can do one round of improvements yourself @disha1202

Fixed indentation and updated markup.
Please take a look.
@dt2patel

**Pull request merged**